cinn/auto_schedule/search_strategy/evolutionary_search.h

```diff
@@ -87,7 +87,13 @@ class EvolutionarySearch {
  private:
   std::vector<SearchState> GetTopKCandidatesFromDatabase(int topk);
 
-  std::vector<SearchState> RandomInitSketch(int num);
+  // Generate sketch as initial population of evolutionary search.
+  // Current optional strategies are "rule_prune" or "random_prune" or "random".
+  // Among them, "random" uses the old method,
+  // "Random_plane" uses the new interface CollectStateTransfer() to simulate the random generation of sketches,
+  // and supports the function of a rule returning multiple SearchStates and random pruning by probability.
+  // "rule_plane" uses rules to prune and generate sketches as efficiently as possible.
```

Comment:
```suggestion
  /**
   * \brief Generate sketch as initial population of evolutionary search.
   * @param strategy ... Current optional strategies are "rule_prune" or "random_prune" or "random".
   * - "random" means uses the old method,
   * - "random_prune" means uses the new interface CollectStateTransfer() to simulate the random generation of sketches, and supports the function of a rule returning multiple SearchStates and random pruning by probability.
  * - "rule_plane" means uses rules to prune and generate sketches as efficiently as possible.
  * @return 
   */
```
修改下格式提升可读性，另外"random"的注释换种说法，old method只对于我们是old，简要说明下策略

Reply:
done